### ISSUE TITLE
Language logistics role

### ISSUE THREAD

**Issue by kgibilterra** [womenwhocodedc/organization#37]
This semi-relates to the "before" section of the meetup [here](https://github.com/womenwhocodedc/organization/blob/master/leadership-resources/Education/how_to_make_a_meetup.md). I was thinking the logistics lead is responsible for the "bigger picture" per month and they would work with @SuWKim to make sure hosts are happy for the month. This frees up the other language leads to focus on making their events successful for that month.

**Logistics Role:**
1. make sure the meetup details are updated for:
   - location
   - topic for the week
   - description
2. make sure hosts & leads for each week know what to expect
   - particular emphasis on the hosts, as they have often been in the dark about our meetups
   - how do people get to the meetup (parking / metro), pictures, passwords, tips, etc
3. schedule a debrief/planning meeting with all active study group leads the day after your hack night
   - discuss who came, what went well, who are you going to follow up with
   - make sure the plan is ready for the following week, and announce that meetup
   - How can we do a better job building our relationships with our sponsors?
   - How can we do a better job building our relationships with our members?
4. assign 1 lead to prepare the topic/activity for each week
   - [Suggested Meeting Types](https://github.com/womenwhocodedc/organization/blob/master/leadership-resources/Education/suggested-meeting-types.md)
   - Hopefully there are enough education leads that everyone can pick one week to be in charge of the event per month, and one week where they are the "backup" and support the other woman
   - the event lead is responsible for working with Laura Phelps & Lauren Jacobson to promote the event / tweet / etc

Ultimately, I don't think this will take more than 1 extra hour a week, but really help with our communication and efficiency throughout the group!

**Comment by SuWKim** [comment]
Coming from me, having this rotating logistics lead for each study groups would be super helpful, and allow us to plan out ahead and preempt last-minute cancellations.  It could also be a way for leads who cannot come to the study groups to be involved for that month by helping to organize.

**Comment by nupurkapoor** [comment]
I agree. This role can basically be the single poc for all study group leads. I especially like the "assign 1 lead to prepare the topic/activity for each week"

**Comment by emgrasmeder** [comment]
I brought this up in the Algorithms channel: I don't think the logistics person needs to necessary _do_ each of these points (for example "update the meetup details"), so long as that task is "delegated" and the responsibility still falls on the logistics person. The example for them is that @aag6z will be making the meetup.com event but Kajari is ultimately responsible for it if Amy doesn't for some reason. 

Would love thoughts on that, though

**Comment by kdmcclin** [comment]
So does the idea of planning for these a couple weeks out mean that the logistics person for, say, October does that two weeks into November, or just whoever's on for the month does what they can for that month, and then whoever's on for the next month does whatever's available and needs to be done for that month.

**Comment by kgibilterra** [comment]
For updating the meetup - the amount of time that it takes to ask someone else to do it, is the same amount of time that it takes to change! (5-15 seconds) So I think it's a better "just update it" task than a delegated task. So if Kajari sees something wrong from Amy's meetup, she should change it and then let Amy know. Basically, I just don't think anyone should ask another person to update the meetup if they know something should be changed. It creates a bottleneck for something that really only takes a couple of seconds.

And idk @kdmcclin - I think monthly might be easier to remember? It's weird this month since we didn't get this message out before it started, so that makes it a little harder for October. (but I don't think too hard?) So say you were putting together the October months. Zuri would start planning for November in the middle of October, but you would still be the point woman for the rest of October. Not sure if that makes sense...

And that would just be so that every lead has experience being in this role. Once everyone has led a month, the responsibility will fall on the newest lead so that it is a sustainable model!

**Comment by kgibilterra** [comment]
Although I did just change it to "make sure the meetup details are updated for" because it does make sense for the responsible lead to update the description. 

I just find the title/location so important/quick to change. It's sometimes not updated for weeks after we know the information because no one feels responsible for it
